server.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+
+const app = express();
+const port = 8080;
+
+app.use(express.json());
+
+const users = [
+  { id: 1, name: 'John Doe', age: 30 },
+  { id: 2, name: 'Jane Smith', age: 25 },
+  { id: 3, name: 'Tom Brown', age: 35 },
+];
+
+app.get('/users', (req, res) => {
+  res.json(users);
+});
+
+app.get('/users/:id', (req, res) => {
+  const id = parseInt(req.params.id, 10);
+  const user = users.find((user) => user.id === id);
+  if (!user) {
+    res.status(404).json({ error: 'User not found' });
+  } else {
+    res.json(user);
+  }
+});
+
+app.post('/users', (req, res) => {
```

Comment:
if we changed it to employee route, we can add validation on the router, for example using [joi](https://joi.dev/api/) and then we the other person just need to plug in the db only and we are done with the backend